File: 01-iniciante/1018/1018.c
#include <stdio.h>

int main()
{
    int n,i;
    int nota100 = 0;
    int nota50 = 0;
    int nota20 = 0 ;
    int nota10 = 0;
    int nota5 = 0;
    int nota2 = 0;
    int nota1 = 0;
    
    scanf("%d",&n); 
    int valorAux = n;
    
    while(valorAux > 0) {
        if (valorAux >= 100){
            nota100 ++;
            valorAux -= 100;
        }
        else if (valorAux>= 50){
            nota50 ++;
            valorAux -= 50;
        }
        else if (valorAux >= 20){
            nota20 ++;
            valorAux -= 20;
        } 
        else if (valorAux >= 10){
            nota10 ++;
            valorAux -= 10;
        } 
        else if (valorAux >= 5){
            nota5 ++;
            valorAux -= 5;
        } 
        else if (valorAux >= 2){
            nota2 ++;
            valorAux -= 2;
        } 
        else if (valorAux >= 1){
            nota1 ++;
            valorAux -= 1;
        } 
    }
   
   printf("%d\n",n);
   printf("%d nota(s) de R$ 100,00\n",nota100);
   printf("%d nota(s) de R$ 50,00\n",nota50);
   printf("%d nota(s) de R$ 20,00\n",nota20);
   printf("%d nota(s) de R$ 10,00\n",nota10);
   printf("%d nota(s) de R$ 5,00\n",nota5);
   printf("%d nota(s) de R$ 2,00\n",nota2);
   printf("%d nota(s) de R$ 1,00\n",nota1);
   
   return 0;
}

File: 01-iniciante/1018/1018.py
n = int(input())
valorAux = n 

nota100 = nota50 = nota20 = nota10 = nota5 = nota2 = nota1 = 0

while valorAux > 0:
    if valorAux >= 100:
        nota100 += 1
        valorAux -= 100
    elif valorAux >= 50:
        nota50 += 1
        valorAux -= 50
    elif valorAux >= 20:
        nota20 += 1
        valorAux -= 20
    elif valorAux >= 10:
        nota10 += 1
        valorAux -= 10
    elif valorAux >= 5:
        nota5 += 1
        valorAux -= 5
    elif valorAux >= 2:
        nota2 += 1
        valorAux -= 2
    else:  
        nota1 += 1
        valorAux -= 1
        
print(f"{n}");
print(f"{nota100} nota(s) de R$ 100,00")
print(f"{nota50} nota(s) de R$ 50,00")
print(f"{nota20} nota(s) de R$ 20,00")
print(f"{nota10} nota(s) de R$ 10,00")
print(f"{nota5} nota(s) de R$ 5,00")
print(f"{nota2} nota(s) de R$ 2,00")
print(f"{nota1} nota(s) de R$ 1,00")